OTE_landing_page.md

```diff
@@ -0,0 +1,80 @@
+# OpenVINO™ Training Extensions {#ote_documentation}
```

Comment:
`{#ote_documentation}` what is this for?

Reply:
it's sphinx tag to build the doc on our website